packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1117,12 +1117,62 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
   }
 
   /** Builds a set of columns based on the given columns mixed with the current overrides. */
-  private _getFixedColumns(newColumns: IColumn[]): IColumn[] {
+  private _getFixedColumns(newColumns: IColumn[], viewportWidth: number): IColumn[] {
```

Comment:
Make sure that when `skipViewportMeasures` is `true` and some columns have both `minWidth`/`maxWidth` *and* a value for `targetWidthProportion`, that the layout 'reverts' to its original fixed-width mode. This will be important for apps which need to support both modes and have a lot of mix/match schema to work with.

Reply:
This should be fixed now